mm2src/mm2_bitcoin/rpc/src/v1/types/transaction.rs

```diff
@@ -2039,4 +2039,44 @@ mod tests {
 
         let _tx: Transaction = serde_json::from_str(json_str).unwrap();
     }
+
+    #[test]
+    fn test_nav_cold_staking_script_pubkey() {
+        let json_str = r#"[
```

Comment:
What do you think, is this JSON long enough to be put to external file? I'm unsure myself 🙂

Reply:
It's not that big actually compared to other JSONs in tests in the same transaction test module, so I will be moving this JSON along the other JSONs to an external file 👌

Reply:
Done